### ISSUE TITLE
Seems to be a bug in the presentation of information 

### ISSUE THREAD

**Issue by yaronelh** [OpenBudget/open-budget-frontend#252]
There is an interactive tutorial you have created demonstrating Corruption suspicions in the settlements budgets,  South division. When you view the graph it shows a blue color and a strange number

Shouldn't this be red ? with the number of additional funding allocated?

![screenshot- 6](https://cloud.githubusercontent.com/assets/7690499/8393109/fe2078ac-1d08-11e5-85c0-6d4394156581.jpg)

**Comment by mushon** [comment]
@yaronelh this is actually ok. There was a ₪429,000 reduction right after the ₪124,000,000 increase:

![change](http://g.recordit.co/KfSMqr0387.gif)

**Comment by yaronelh** [comment]
I get it now,,, :)   
Looks like the same line, the reduction is almost invisible.

**Comment by mushon** [comment]
True :(
I’m open to suggestions there

**Comment by yaronelh** [comment]
I would go with two things, first I would leave the number in red when The number is negative in general. because it still red, no matter the small amount redacted.

Second point I would consider replacing the usage of  +, - , =
Which is a little misleading because it doesn't have the visual / psychological weight necessary.
since there isn't a problem of room there, I would think about adding what it is in text

Here's a quick mockup of my thoughts (Okay not so quick but anyway...)
![untitled-1](https://cloud.githubusercontent.com/assets/7690499/8395755/e8f96fd6-1d8c-11e5-9502-2078490dd89c.jpg)

**Comment by mushon** [comment]
רעיון טוב:
![screen shot 2015-06-28 at 12 12 43 pm](https://cloud.githubusercontent.com/assets/27123/8395784/137daaae-1d8f-11e5-88dc-adb6d3493750.png)

**Comment by yaronelh** [comment]
Below I've added two comments, there is no judgment here
in fact you did an amazing job of doing something, while most will be content in just talking about doing.

The equals still doesn't work for me, for some reason it's like it's not giving me the right meaning that this is a summary number underneath the equals sign 
Maybe because an equals sign is not usually used row after row and the detachment is causing the problem, It creates a cognitive load, something you need to process before you can understand it. When you can just give the user the answer without the user needing to think.


I know that the word תוספת is the official word of what has happened. but I used the word מעבר because It adds another layer above the technical understanding, The usage of מעבר as a reminder that there was an initial budget, still keeps the same meaning.

מעבר לתקציב 2014

Good thing it's Sunday, not a full work day :)

**Comment by yonatann** [comment]
I agree about using מעבר and removing the equal sign, feels less technical and I like it.

**Comment by mushon** [comment]
here we go…
![new](http://g.recordit.co/xVSY7e6ddY.gif)

**Comment by yaronelh** [comment]
Looks a lot clearer, 
The only thing I would say in general, is that there is a lot of colors switching. you could run into a problem of "running out of colors" for separation. I would review it or give it another thought before that happens or before color coding becomes less effective due to the amount of variations.

Also, when the color is in the bounding box it has one level of separation between it and the numbers, don't know if you get my drift. But in general this is a hard problem to solve.

**Comment by mushon** [comment]
@yaronelh not sure I get your drift :/

**Comment by yaronelh** [comment]
The second one is less clear what I meant, I'll try to rephrase

I've made an example of what I meant about, the colors and degree of separation. 
Basically the idea Is that when the color is pronounced in the bounding box, It is one degree separated from the key information (splitting the focus).

Plus there is a problem of hierarchy of information, with the current one your eye can't focus on a specific point it is hard to flow between the points of information.

It takes to things into consideration, color separation, and leading the eye to key information. 

![artboard 1 copy](https://cloud.githubusercontent.com/assets/7690499/8442832/18544cca-1f8a-11e5-9c8f-8f29ef28310b.jpg)
![artboard 1](https://cloud.githubusercontent.com/assets/7690499/8442833/1856766c-1f8a-11e5-9f62-14eb553d3e58.png)

** This is just an example (for the discussion), I haven't had the time to go over the complete color coding options and variations to see that this particular solution works, Although I think it could work with the whole gradient I dear of cold colors to warm colors , It continues the same line but in a more subtle way.

**Comment by mushon** [comment]
ok @yaronelh, I think I understand where you were going with this. 
I agree this works much better:

![check it now](http://g.recordit.co/irOwrOmfeu.gif)

**Comment by yaronelh** [comment]
Well not exactly, what I'm saying is that there are too many signals that use color as indicators, to the point that color cues are not really understandable anymore. This example has even more colors

As I understand it from the instructions on the homepage, more expenses mean red color reduced expenses are in the blue. that works excellent when it's on a Circles graph like on the homepage.

But within the tooltips that are explanations for the current item it becomes unclear, you no longer need to remember the simple 1 to 1 color signal, sometimes Inside a tooltip there are 4 items each with its own color making it unreadable.

In my opinion colors are a secondary indicator, but in the current set up it acts as the main course.

We need to turn the hierarchy of information back to first present the information clearly and then add additional layers of information like colors.

What I suggested might not be the best solution, it's just one way of toning down the amount of colors and bringing out the information which is more important.

To create a proper taste case I need to know all the possible variations on this graph
and create a version that works across  all of them

First we need to agree that there is a problem with this form of presentation, if we are in agreement then we can find a solution.

Cheers
